emc2/simulator/radar_moments.py

```diff
@@ -896,7 +896,7 @@ def _calc_sigma_d_tot_cl(tt, N_0, lambdas, mu, instrument,
     num_diam = len(p_diam)
     Dims = Vd_tot.shape
     for k in range(Dims[2]):
-        if np.all(total_hydrometeor[tt, k] == 0):
+        if np.all(total_hydrometeor[0, tt, k] == 0): #++MZ total_hydrometeor[tt, k] == 0):
```

Comment:
@mengz1993 , since we are interested in skipping an iteration here in case we do not have subcolumns, this line should be checking:
`np.all(total_hydrometeor[:, tt, k] == 0)`